rnd/autogpt_server/autogpt_server/cli.py

```diff
@@ -109,5 +179,43 @@ def event():
     print("Event sent")
 
 
+@test.command()
+@click.argument("server_address")
+@click.argument("graph_id")
+def websocket(server_address: str, graph_id: str):
+    """
+    Tests the websocket connection.
+    """
+    import asyncio
+
+    import websockets
+
+    from autogpt_server.server.ws_api import ExecutionSubscription, Methods, WsMessage
+    import websockets
+
+    from autogpt_server.server.ws_api import ExecutionSubscription, Methods, WsMessage
+
+    async def send_message(server_address: str):
+        uri = f"ws://{server_address}"
+        async with websockets.connect(uri) as websocket:
+            try:
+                msg = WsMessage(
```

Comment:
@aarushik93 Here is how you subscribe to a graph for execution events.

You get this response:

`{"method":"subscribe","data":null,"success":true,"channel":"133475d7-c610-4384-b7be-9786ff54e31e","error":null}`

Then you get "updates" as the graph executes:

```
{"method":"update","data":{"graph_id":"b9cdf471-4558-4582-be08-d1066a1c1529","graph_exec_id":"d1313f24-bacd-451f-bf28-0a851437b218","node_exec_id":"9665ff72-62e9-4653-861d-589479644fa6","node_id":"95d6f88f-e0da-4558-8cd1-8645707d6efe","status":"QUEUED","input_data":{"input":"Hello, World!"},"output_data":{},"add_time":"2024-07-05T09:37:10.972000Z","queue_time":"2024-07-05T11:37:10.975000Z","start_time":null,"end_time":null},"success":null,"channel":"b9cdf471-4558-4582-be08-d1066a1c1529","error":null}
Response from server: {"method":"update","data":{"graph_id":"b9cdf471-4558-4582-be08-d1066a1c1529","graph_exec_id":"d1313f24-bacd-451f-bf28-0a851437b218","node_exec_id":"9665ff72-62e9-4653-861d-589479644fa6","node_id":"95d6f88f-e0da-4558-8cd1-8645707d6efe","status":"RUNNING","input_data":{"input":"Hello, World!"},"output_data":{},"add_time":"2024-07-05T09:37:10.972000Z","queue_time":"2024-07-05T11:37:10.975000Z","start_time":"2024-07-05T11:37:10.983000Z","end_time":null},"success":null,"channel":"b9cdf471-4558-4582-be08-d1066a1c1529","error":null}
```